holmes.py

```diff
@@ -267,6 +280,37 @@ def alertmanager(
     if json_output_file:
         write_json_file(json_output_file, results)
 
+
+@generate_app.command("alertmanager-tests")
+def generate_alertmanager_tests(
+    alertmanager_url: Optional[str] = typer.Option(None, help="AlertManager url"),
```

Comment:
correct me if im wrong but it looks like for this command alertmanager_url is not optional

Reply:
It's still optional here, because it could be defined in the config file.

Reply:
than specifically in this function I would check before the line
    source = config.create_alertmanager_source()
to make it easier for users
![Screen Shot 2024-06-23 at 11 34 03](https://github.com/robusta-dev/holmesgpt/assets/97387909/9bf6daf5-6643-4845-a5ff-788f67da4a19)

Reply:
also a different issue here, want to make sure you dont miss it
https://github.com/robusta-dev/holmesgpt/pull/56#issuecomment-2184900236

Reply:
I think it's ok here. The last line of output clearly says `--alertmanager-url must be specified`.
If someone misses it or is confused I'll rework it later.